utils/block-nameify.ts

```diff
@@ -1,10 +1,11 @@
-import preFormattedBlockNames from './preformatted-block-names.json';
+import preFormattedBlockNames from './preformatted-block-names.json' assert { type: 'json' };
```

Comment:
This doesn't seem to be necessary.  If I remove the `assert`, TS is still capable of inferring the types correctly.

My understanding was that `assert` is a security measure that prevents non-json files from being evaluated. Given that we control these files, do we need this?

Reply:
This is probably me not building the config/utils, and then getting the `any` error 🤦‍♂️